.github/workflows/prettier.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  schedule:
+    # https://crontab.guru/#0_0_*_*_7
+    - cron: "0 0 * * 7"
```

Comment:
@gr2m This value doesn't seem to be liked by GitHub Actions.
I used Crontab Guru to create it
```
invalid `cron` attribute "0 0 * * 7"
```